homeassistant/components/vera/__init__.py

```diff
@@ -171,6 +175,19 @@ def should_poll(self):
         """Get polling requirement from vera device."""
         return self.vera_device.should_poll
 
+    @property
+    def device_info(self):
```

Comment:
Device info requires a config entry to be used. Please submit this PR after the config entries PR.

https://developers.home-assistant.io/docs/en/device_registry_index.html#defining-devices